src/test/plots/profile_statistics_plots/profile_statistics_plots_test.py

```diff
@@ -1,9 +1,10 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
 import os
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group